mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -783,4 +789,51 @@ mod tests {
             panic!("Should not reach here");
         }
     }
+
+    fn test_bridge_settings() -> BridgeSettings {
```

Comment:
since we moved "how we read settings" logic from `mqtt-bridge` to `mqtt-edgehub`, here we just create test data instead of reading it from `tests/config.json`